src/sentry/integrations/pipeline.py

```diff
@@ -33,6 +35,10 @@ class IntegrationPipeline(Pipeline):
     pipeline_name = "integration_pipeline"
     provider_manager = default_manager
 
+    def get_analytics_entry(self) -> Optional[PipelineAnalyticsEntry]:
```

Comment:
Let's definitely try and pull this out into it's own PR for moving the analytics stuff

Reply:
Done: https://github.com/getsentry/sentry/pull/27176